### DISCUSSION TITLE
[Desktop] Unable to add tabulation

### DISCUSSION THREAD

**Discussion by DmitrySboychakov** [singerdmx/flutter-quill#396] [Desktop]
My issue is about [Desktop]
I have tested it on the MacOS
There is no reaction to the Tab button on keyboard, while expected the \t symbol added to the text

**Comment by DmitrySboychakov** [comment]
@singerdmx what do you think will be the best way to fix that? 
Adding the new ShortCut with keyTab is one of option, but it doesn't looks good to me. What is your opinion?

**Reply by singerdmx** [reply]
Let me use Quill JS for example:

`[{"insert":"\tx\n\tb\n"}]`

The above data is rendered as follows:

![image](https://user-images.githubusercontent.com/122956/133196618-0524c39a-2d09-4669-8a34-1cfe01abd84c.png)

**Reply by szetakyu** [reply]
When I press Tab in flutter quill, it is just like a short white space 
![image](https://github.com/singerdmx/flutter-quill/assets/41570451/8b4b69a1-52bc-409d-a6b9-99212e623d16)
![image](https://github.com/singerdmx/flutter-quill/assets/41570451/72a575ab-0b08-4675-a281-dbbaaa09e47d)
